Assets/ScriptableObjects/Plugin Scripts/FragilePlugin.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu]
+public class FragilePlugin : BlockPlugin
+{
+    public float BreakAfterTime;
+
+    public override void OnUpdate()
+    {
+        if (_block.GetIsTranslating() || _block.GetIsPlayerTouched())
+        {
+            Destroy(_block.gameObject, BreakAfterTime);
+        }
```

Comment:
The block is destroyed as long as the player touches it? (Not standing on top?)

Reply:
Ah whoops, my bad. Right now I'm planning to add a box collider to the top of the gravel block, and use OnTriggerEnter/Exit to determine if the player is standing on it.

Reply:
For now it's just the gravel block. Would it be better if every block has a box collider on top of it (then future blocks can use OnTrigger)?